### ISSUE TITLE
Middleman init issue on Windows 

### ISSUE THREAD

**Issue by trejster** [middleman/middleman#1724]
I have installed the latest version of ruby 2.2.3 and the associated dev kit which all installed successfully on a windows 8 pc. Installed middleman which installed successfully. Now on running middleman init I am getting an error:

"c:/Ruby22/bin/middleman:23 in 'load': cannot load such file -- C:/Ruby22/lib/ruby/gems/2.2.0/gems/middleman-core-4.0.0/bin/middleman (LoadError).."

Can anyone help please?

Looking in the quoted directory there is no bin directory underneath middleman-core-4.0.0?

**Comment by hovancik** [comment]
in Unix world I would try running `bundle exec middleman init`

**Comment by katafrakt** [comment]
> in Unix world I would try running `bundle exec middleman init MY_PROJECT`

...which would probably not work because you usually don't have `Gemfile` in the directory where you run `middleman init` (at least I don't). Also, `bundle exec` works fine on Windows.

Regarding the problem, I had similar issue not long ago on Linux. I _think_ I solved it by reinstalling `middleman-cli` gem, but I'm not sure about it. Can you give it a try? If it does not work, can you post a result of running `gem list | find "middleman"`?

**Comment by rgb-one** [comment]
What version of middleman are you using?

**Comment by trejster** [comment]
This is using latest version of middleman. I have just tried uninstalling all middleman gems and reinstalling (gem install middleman). The install succeeds.

I then try and create a new site named 'site:
 'middleman init test'

I get the exact same error...

I then tried using command 'bundle exec middleman init test'. Which worked

However if I then try and run middleman within my new directory (test). I get the exact same error mentioned above.

Also tried running bundle exec middleman, got a different error this time:

"could not find gem 'middleman-livereload'...Run bundle install to install missing gems"

So I run 'bundle install'. It works and I can now view the site - so whats going on here? Any idea why I have to follow this process to get it to work?

**Comment by hovancik** [comment]
is `middleman-livereload` in Gemfile?

**Comment by trejster** [comment]
This was a brand new site so there was no gemfile to start with, it should create the gemfile on calling init shouldnt it? I can see that the gemfile that has been created in the new site does have middleman-livereload.

What I dont understand is why I have to keep using the 'bundle exec' prefix to use middleman?

**Comment by dg-ratiodata** [comment]
Using `bundle exec` only makes sense after you initialized your site.

**Comment by tdreyno** [comment]
@trejster Because Ruby is awful in some ways and the dependency management framework Bundler is not built-in :(

**Comment by kaishiro** [comment]
Actually running into a similar issue - Ruby 2.3.0 / Middleman 4.0.0

Fresh install on OSX, rbenv/ruby-build via homebrew.  
Vanilla `gem install middleman` pulls down 4.0.0.  
Upon running `middleman init` I get:

```
/Users/kaishiro/.rbenv/versions/2.3.0/bin/middleman:23:in `load': cannot load such file -- /Users/kaishiro/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/middleman-core-4.0.0/bin/middleman (LoadError)
	from /Users/kaishiro/.rbenv/versions/2.3.0/bin/middleman:23:in `<main>'
```

touch a `.Gemfile` with just one line: `gem 'middleman'`, then `bundle install` and `bundle exec middleman init` works fine.  Was assuming it was just me since I'm new to OSX from Linux and am still learning the whole homebrew system.

**Comment by IvonaBarun** [comment]
Have you ever got question 'Do you want use Compass?' after middleman init on command line (windows)? I can't figure out what to do to continue, thanks!

**Comment by kaishiro** [comment]
@IvonaBarun You should just be able to type `y` or `n`, depending on if you want to use Compass or not.  Are you getting a specific error after responding?

**Comment by IvonaBarun** [comment]
@MatthewWhite thanks for respond. I've tried replaying y and n but nothing
has happened. Here is print-screen attached.

**Comment by jawwadzafar** [comment]
I was having problem with running middleman on **windows** you can use this template:

https://github.com/jawwadzafar/middleman-boilerplate.git

**Comment by linjer** [comment]
I am able to reproduce this issue. Simply using ruby-2.3.0 via RVM and doing `gem install middleman` leads to `/Users/jrlin/.rvm/gems/ruby-2.3.0/bin/middleman:23:in `load': cannot load such file -- /Users/jrlin/.rvm/gems/ruby-2.3.0/gems/middleman-core-4.1.6/bin/middleman (LoadError)`

It appears gem install middleman doesn't install dependencies.

Switching to ruby-2.2.4, `gem install middleman` installs a bunch more dependencies and `middleman init` works fine.

**Comment by tdreyno** [comment]
whoa, this is a ruby bug?

Does `/Users/jrlin/.rvm/gems/ruby-2.3.0/bin/middleman` not exist?

**Comment by linjer** [comment]
@tdreyno probably. The issue had to do with `middleman-core` and various other dependencies not installing.

It seems that the issue has been resolved on my side. I simply killed my `ruby-2.3.0`, reinstalled, including `gem install middleman` and it now correctly pulls in all dependencies, which was not being done at the time of my previous error.

I'm now able to successfully do `middleman init test` at this time with middleman `4.1.6`. 

As a note, I had done the same steps above before reporting the issue 4 days ago.

**Comment by sandstrom** [comment]
I'm doing some issue-gardening 🌿🌷🌾 and came across this issue.

Sorry that this has lingered for quite a while! I'd suggest closing this issue. If the problem resurfaces a repro or failing test would be a good start in solving this. Ideally also a patch with a fix.

**Comment by jessicaldale** [comment]
Had the same issue. as per @linjer comment, switching to ruby-2.2.4, gem install middleman fixed the problem. Uninstalled the existing middleman gems first just as a precaution.

**Comment by coreyward** [comment]
Ran into this issue after not using Middleman for a while. Seems the fragmentation between `middleman-core`, `middleman-cli`, and `middleman` creates a host of issues. Running `gem install middleman` did not help, nor did uninstalling the `middleman` gem before reinstalling, or reinstalling `middleman-core`. I had to remove and reinstall `middleman-cli`, including letting `gem` remove the `middleman` executable.
